03Lesson/01Homework/index.html

```diff
@@ -1,10 +1,23 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <meta charset="utf-8">
+  <title>Berries</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-
+<table>
+  <tr>
+    <td>
+      <div>
+      <img src="img/berries.jpg" alt="berries" />
+      </div>
+    </td>
+    <td>
+      <div class="berries">
+        Berries
+      </div>
+    </td>
+</table>
```

Comment:
This is probably work in progress? Check the indentation: `img` inside of `div` needs to be indented. Missing closing `tr` tag.